release/serve_tests/workloads/anyscale_service_utils.py

```diff
@@ -0,0 +1,113 @@
+import boto3
+from contextlib import contextmanager
+import logging
+import os
+from typing import Dict, List, Optional, Union
+
+from anyscale.authenticate import AuthenticationBlock
+from anyscale import AnyscaleSDK, service
+from ray._private.test_utils import wait_for_condition
+
+
+logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+
+class DeferredEnvVar:
+    def __init__(self, var: str, default: Optional[str] = None):
+        self._var = var
+        self._default = default
+
+    def __str__(self):
+        return os.environ.get(self._var, self._default)
+
+
+RELEASE_AWS_ANYSCALE_SECRET_ARN = DeferredEnvVar(
+    "RELEASE_AWS_ANYSCALE_SECRET_ARN",
+    "arn:aws:secretsmanager:us-west-2:029272617770:secret:"
+    "release-automation/"
+    "anyscale-token20210505220406333800000001-BcUuKB",
+)
+
+
+def get_anyscale_cli_token() -> str:
+    try:
+        token, _ = AuthenticationBlock._load_credentials()
+        logger.info("Loaded anyscale credentials from local storage.")
+        return token
+    except Exception:
+        pass  # Ignore errors
+
+    logger.info("Missing ANYSCALE_CLI_TOKEN, retrieving from AWS secrets store")
+    return boto3.client("secretsmanager", region_name="us-west-2").get_secret_value(
+        SecretId=str(RELEASE_AWS_ANYSCALE_SECRET_ARN)
+    )["SecretString"]
```

Comment:
is this the standard way to do this in the release tests? is there some shared util we can use from other release tests?

Reply:
Yeah this logic is duplicated in `ray_release/aws.py`, but that's not built into the ray wheel. @aslonnie what would be a good place to put this logic as a shared util?

Reply:
Update: I've removed this because it is unnecessary.